frontend/src/pages/RecipientRecord/pages/Monitoring/components/SpecialistTags.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tooltip from '../../../../../components/Tooltip';
+
+export default function SpecialistTags({ specialists }) {
+  return specialists.map((specialist) => {
+    if (!specialist.name) return null;
+
+    return (
+      <p className="usa-prose margin-top-0 margin-bottom-1 margin-right-1 bg-base-lightest radius-md padding-x-1 display-inline-flex flex-align-center flex-justify-between text-decoration-underline">
```

Comment:
Seeing a unique key warning on the `p` tag:

```suggestion
import { v4 as uuidv4 } from 'uuid';
import Tooltip from '../../../../../components/Tooltip';

export default function SpecialistTags({ specialists }) {
  return specialists.map((specialist) => {
    if (!specialist.name) return null;

    return (
      <p key={`specialist-${uuidv4()}`} className="usa-prose margin-top-0 margin-bottom-1 margin-right-1 bg-base-lightest radius-md padding-x-1 display-inline-flex flex-align-center flex-justify-between text-decoration-underline">
```

Or whatever makes more sense

Reply:
thanks